game-rimworld/index.js

```diff
@@ -9,16 +9,21 @@ function findGame() {
 }
 
 function main(context) {
+  var arch = process.platform === 'x64';
```

Comment:
`process.platform` returns operating system platform and therefore will always resolve as 'win32' on Windows systems.

I'm thinking you probably meant to use `process.arch`

So `const arch = process.arch === 'x64'`  would fix this problem.

That being said, keep in mind that Vortex only supports 64bit systems so this check is kind of redundant as you can just assume that whoever is modding the game using Vortex should always use RimWorldWin64.exe